sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/utils/AvroUtilsTest.java

```diff
@@ -49,15 +59,14 @@
   public void supportsAnyAvroSchema(
       @From(RecordSchemaGenerator.class) org.apache.avro.Schema avroSchema) {
     // not everything is possible to translate
-    assumeThat(avroSchema, not(containsField(AvroUtilsTest::hasArrayOrMapOfNullable)));
     assumeThat(avroSchema, not(containsField(AvroUtilsTest::hasNonNullUnion)));
 
-    Schema schema = AvroUtils.toSchema(avroSchema);
+    Schema schema = AvroUtils.toBeamSchema(avroSchema);
     Iterable iterable = new RandomData(avroSchema, 10);
     List<GenericRecord> records = Lists.newArrayList((Iterable<GenericRecord>) iterable);
 
     for (GenericRecord record : records) {
-      AvroUtils.toRowStrict(record, schema);
+      AvroUtils.toBeamRowStrict(record, schema);
```

Comment:
Added this test. It turned up a pre-existing bug converting Avro records to Beam rows caused because ByteBuffer is mutable (this case was handled properly for FIXED but not for BYTES).